dataset_examples/dataset_register_cloud.py

```diff
@@ -36,20 +36,22 @@
 # STEP 3
 # Create final dataset object formed from cloud upload path
 # Select the x_col_names which are the features and the y_name as the target while registering the dataset
+
+# Note: Populate the fields ["<SELECTED FEATURE 01", "<SELECTED FEATURE 02"], <TARGET>, s3://<path_to_train_dataset>
 data_set = DataSet.from_cloud(
     df_id=df_response.df_id,  # data family id
-    x_col_names=["tweet"],  # feature columns
-    y_name="sentiment",  # target column
+    x_col_names=["<SELECTED FEATURE 01", "<SELECTED FEATURE 02"],  # feature columns
+    y_name="<TARGET>",  # target column
     delimiter=",",  # delimiter used for the input dataset segments
     name="CloudUploadSDK_01",  # dataset name (should be unique)
     data_category=DataCategory.Text,
     # supply dataset category (Text data category) (Text: If any of the feature column is text)
     credential_id=cred_id,  # pass cred id to access the s3
     # path to the dataset segment in s3 bucket, keep none if unsplit
-    train_source="s3://path_to_dataset/twitter_train.csv",
+    train_source="s3://<path_to_train_dataset>",
     # path to the dataset segment in s3 bucket, keep none if unsplit
-    test_source="s3://path_to_dataset/twitter_test.csv",  # path to the dataset segment in s3 bucket
-    unsplit_source="s3://path_to_dataset/twitter_unsplit.csv"  # set path here if your dataset is unsplit
+    test_source="s3://<path_to_test_dataset>",  # path to the dataset segment in s3 bucket
+    unsplit_source="s3://<path_to_dataset>",  # set path here if your dataset is unsplit keep None if train or test is supplied
 )
 
```

Comment:
no option for validate , along with test path and train path ?

Reply:
its there will add it